src/core/core.tooltip.js

```diff
@@ -806,17 +809,17 @@ module.exports = function(Chart) {
 				this.drawBackground(pt, vm, ctx, tooltipSize, opacity);
 
 				// Draw Title, Body, and Footer
-				pt.x += vm.xPadding;
+				// pt.x += vm.xPadding;
 				pt.y += vm.yPadding;
 
 				// Titles
-				this.drawTitle(pt, vm, ctx, opacity);
+				this.drawTitle(pt, vm, ctx, opacity, tooltipSize);
```

Comment:
Not sure this is the best solution. `tooltipSize` is just `vm.width` and `vm.height` (line 792). I would prefer just using `vm.width` and `vm.height` inside these functions since it avoids another parameter